Makefile

```diff
@@ -89,13 +78,40 @@ ifeq ($(TRAVIS_BRANCH),production)
 endif
 
 .PHONY: travis_push
-travis_push:: banner ensure build deploy
+travis_push::
+	$(MAKE) banner
+	$(MAKE) ensure
+ifeq ($(TRAVIS_BRANCH),master)
+	export HUGO_BASEURL = https://staging.pulumi.io/
```

Comment:
When running `hugo` to build the site, if we're on the master branch, I want to specify the `HUGO_BASEURL` env variable so that `https://staging.pulumi.io/` is used for the base URL instead of the default `https://pulumi.io/` specified in `config.toml`. I *think* exporting it like this here before calling the `build` target below will work (the `hugo` command part of the `build` target needs to see the variable, but I haven't confirmed.

Repeating this for each `travis_` target is pretty lame. I'm sure there's a much more elegant way of handling this, and would love any suggestions, although, this Makefile isn't long for this world...

Reply:
Yeah, [doesn't work](https://travis-ci.com/pulumi/docs/builds/112964262).

```
export HUGO_BASEURL = https://staging.pulumi.io/
/bin/sh: 1: export: : bad variable name
make: *** [travis_pull_request] Error 2
The command "make travis_${TRAVIS_EVENT_TYPE}" exited with 2.
```

Will fix.